README.md

```diff
@@ -139,13 +139,35 @@ jobs:
           commit_sha: ${{ env.GITHUB_COMMIT_SHA }}
 ```
 
+#### Uploading a component using a Github OIDC Token
```

Comment:
This PR changes the recommended flow of the action use. 
So it's important to explain what OIDC tokens are (add links to GitHub Docs) and how to configure the component in the registry.

This example should be the first one in the list.

Also the description around line 9 must be updated.